### ISSUE TITLE
Clarity on passes/fails for SC 3.2.7 Visible Controls

### ISSUE THREAD

**Issue by mraccess77** [w3c/wcag#2100]
Situation 1 - you have an image - when you hover over the image the pointer shows a plus sign - when you click the image, an image pops up in a zoomed view.  When you tab to the smaller image a button becomes visible letting you know you can zoom in. 

Does this fail or pass the criteria?  We do have an exception for "enhance the experience for keyboard navigation;" but this doesn't seem to fit for this situation as it's the only way to access the zoomed image with the keyboard.  An example of this can be found in Articulate Rise courses.

Situation 2 - the understanding doc states "User interface components can be available through other visible   ​components such as sub-menus, edit buttons, tabs, or thumbnails of media."

If a button is used to make more things available that would be available on hover/focus- should we assume that a button that is only text without a border or icon indicating something else would fail and that a button with a border would pass?  Seems like a button without a border but with an icon like a down arrow would also pass.   Would a button border be sufficient as a clue?
